mkosi/__init__.py

```diff
@@ -1693,7 +1705,7 @@ def make_image(state: MkosiState, skip: Sequence[str] = [], split: bool = False)
                 )
 
             esp = (state.config.bootable == ConfigFeature.enabled or
-                  (state.config.bootable == ConfigFeature.auto and bootloader.exists()))
+                  (state.config.bootable == ConfigFeature.auto and bootloader and bootloader.exists()))
```

Comment:
Not introduced here, but the indentation is a bit confusing, either
```python
            esp = (state.config.bootable == ConfigFeature.enabled or
                   (state.config.bootable == ConfigFeature.auto and bootloader and bootloader.exists()))
```
or
```python
            esp = (
                state.config.bootable == ConfigFeature.enabled or
                (state.config.bootable == ConfigFeature.auto and bootloader and bootloader.exists())
            )
```
would be clearer.